Index: src/test/java/at/ac/fhcampuswien/FieldTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package at.ac.fhcampuswien;\r\n\r\nimport org.junit.jupiter.api.Test;\r\nimport org.springframework.ui.Model;\r\nimport org.springframework.validation.support.BindingAwareModelMap;\r\nimport static org.junit.Assert.*;\r\n\r\npublic class FieldTest {\r\n\r\n    @Test\r\n    public void test1() {\r\n       Field myField = new Field();\r\n       myField.createField();\r\n       char[][] tmpField;\r\n       tmpField = new char[10][10];\r\n       Assert.assertTrue(tmpField == myField.getField());\r\n\r\n    }\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n}\r\n
===================================================================
diff --git a/src/test/java/at/ac/fhcampuswien/FieldTest.java b/src/test/java/at/ac/fhcampuswien/FieldTest.java
--- a/src/test/java/at/ac/fhcampuswien/FieldTest.java	
+++ b/src/test/java/at/ac/fhcampuswien/FieldTest.java	
@@ -1,8 +1,8 @@
 package at.ac.fhcampuswien;
 
 import org.junit.jupiter.api.Test;
-import org.springframework.ui.Model;
-import org.springframework.validation.support.BindingAwareModelMap;
+//import org.springframework.ui.Model;
+//import org.springframework.validation.support.BindingAwareModelMap;
 import static org.junit.Assert.*;
 
 public class FieldTest {
